src/appleseed/renderer/modeling/material/genericmaterial.cpp

```diff
@@ -102,6 +102,14 @@ namespace
                     context.get());
             }
 
+            if  (m_render_data.m_volume && has_alpha_map())
+            {
+                RENDERER_LOG_WARNING(
+                    "%smaterial is assigned with alpha map and volume at the same time: "
```

Comment:
*is assigned an*

Replace colon by semicolon, again in the name of consistency with the other similar message.

Reply:
Also, _a volume_.